x/twap/types/utils.go

```diff
@@ -62,6 +60,12 @@ func LexicographicalOrderDenoms(denom0, denom1 string) (string, string, error) {
 	return denom0, denom1, nil
 }
 
+// CanonicalTimeMs returns the canonical time in milleseconds used for twap
+// math computations in UTC. Removes any monotonic clock reading prior to conversion to ms.
+func CanonicalTimeMs(twapTime time.Time) int64 {
```

Comment:
just curious, why is it called "canonical" time?

Reply:
Canonical with respect to block times. It strips away any mononic part from the clock and returns the time in UTC,

This should already be done given that we use `ctx.BlockTime()`. However, this is useful for tests since we might not be setting up the time correctly in tests